webapp/src/main/java/com/box/l10n/mojito/service/ai/translation/AITranslationFilterConfiguration.java

```diff
@@ -0,0 +1,59 @@
+package com.box.l10n.mojito.service.ai.translation;
+
+import java.util.Map;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ConfigurationProperties(prefix = "l10n.ai.translation.filter")
+public class AITranslationFilterConfiguration {
+
+  private Map<String, RepositoryConfig> repositoryConfig;
+
+  public Map<String, RepositoryConfig> getRepositoryConfig() {
+    return repositoryConfig;
+  }
+
+  public void setRepositoryConfig(Map<String, RepositoryConfig> repositoryConfig) {
+    this.repositoryConfig = repositoryConfig;
+  }
+
+  public static class RepositoryConfig {
+    private boolean excludePlurals;
+    private boolean excludePlaceholders;
+    private boolean excludeHtmlTags;
+    private String excludePlaceholdersRegex = "\\{[^\\}]*\\}";
+
+    public boolean isExcludePlurals() {
```

Comment:
naming nit/potentially subjective, for the three is* methods: might improve readability to prefix them with should, e.g.: "shouldExcludePlurals" ?